tags/tabs/tabs.talon

```diff
@@ -0,0 +1,38 @@
+tag: user.tabs
+-
+
+# Creation
+tab (open | new): app.tab_open()
+tab (open | new) named [<user.text>]: user.tab_open_with_name(text or "")
+tab (reopen | restore): app.tab_reopen()
+tab (clone | duplicate): user.tab_clone()
+
+# Destruction
+tab close: user.tab_close_wrapper()
+tab close all: user.tab_close_all()
+tab (close others | solo): user.tab_close_others()
+
+# Navigation
+[go] tab (last | previous): app.tab_previous()
+[go] tab next: app.tab_next()
+[go] tab <number>: user.tab_focus_index(number)
```

Comment:
The optional `go` causes an issue here if numbers are not prefixed and you are trying to insert a tab character followed by a literal number. The optional go in other cases seems to work well.